### PR TITLE
Chapter_8 : BASIC LOGIN

### PR DESCRIPTION
**Pull request by thunv-1049**

### CODE REVIEW

**Comment by thunv-1049** [comment]
Screenshot from 2020-09-10 16-23-30

**Comment by thunv-1049** [comment]
ready

File: app/assets/stylesheets/sessions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
```
Comment (chungpv-1008) [inline app/assets/stylesheets/sessions.scss:1]: remove this file

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+        <% if logged_in? %>
+        <li><%= link_to t(".users"), '#' %></li>
+        <li class="dropdown">
+         <%= link_to "#", class: "dropdown-toggle",
+              "data-toggle": "dropdown" do %>
+              <%= t ".account" %> <b class="caret"></b>
+            <% end %>
+          <ul class="dropdown-menu">
+            <li><%= link_to t(".profile"), current_user %></li>
+            <li><%= link_to t(".settings"), '#' %></li>
```
Comment (chungpv-1008) [inline app/views/layouts/_header.html.erb:18]: dùng " nhé a

**Review by chungpv-1008 — COMMENTED** [review]
Fix rồi tự merge nhé a

File: app/views/layouts/_header.html.erb (hunk 2 above)
Comment (thunv-1049) [reply app/views/layouts/_header.html.erb:18]: Vâng anh

**Pull request merged**